data/remnant ships.txt

```diff
@@ -348,6 +347,87 @@ ship "Gull" "Gull (Bunks)"
 
 
 
+ship "Ibis"
+	sprite "ship/ibis"
+	thumbnail "thumbnail/ibis"
+	attributes
+		category "Medium Warship"
+		licenses
+			"Remnant Capital"
+		"cost" 30000000
```

Comment:
This is 50% more expensive than the hull cost of the Albatross. Seems a bit much given that this is a medium warship.

Reply:
Reduced to a bit lower than the hull price of an Albatross.